### ISSUE TITLE
Separate button examples

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#1795]
The [button design pattern](https://w3c.github.io/aria-practices/#button) has two examples:

1. [Button Examples](https://w3c.github.io/aria-practices/examples/button/button.html)
2. [Button Examples (IDL)](https://w3c.github.io/aria-practices/examples/button/button_idl.html)

Both of these include two types of button: a stateless push-button ("Print Page"), and a toggle button using `aria-pressed` ("Mute").  The APG includes some other types of button, notably disclosure, menu button and spinbutton, and it's not clear why the toggle button has been added to this page with the standard stateless one rather than being broken out into its own pattern.

CC @mcking65 - applies to ARIA-AT testing.

**Comment by jscholes** [comment]
Also CC @sinabahram

**Comment by mcking65** [comment]
@jscholes 

I agree that they should be separated. Button, grid, listbox, and I think one other pattern all have example pages with multiple examples. The redesign project may be the force that finally gets us to split them up.

Since we still have manual copy and adjustment to the examples for aria-at, this is not a blocker for your work, right?

**Comment by jscholes** [comment]
@mcking65 

> Since we still have manual copy and adjustment to the examples for aria-at, this is not a blocker for your work, right?

This should be fine for now.
